packages/sheets-drawing/src/controllers/sheet-drawing.controller.ts

```diff
@@ -66,6 +66,7 @@ export class SheetsDrawingLoadController extends Disposable {
                 },
                 onLoad: (unitId, value) => {
                     this._sheetDrawingService.registerDrawingData(unitId, value);
+                    this._drawingManagerService.registerDrawingData(unitId, value);
```

Comment:
Why should we register the value twice on different service? IMHO, `_sheetDrawingService` should do this.

Reply:
I don't know why, but this was actually 2 service to manager drawing

Reply:
> Why should we register the value twice on different service? IMHO, `_sheetDrawingService` should do this.

I don't know why, but this was actually 2 service to manager drawing

Reply:
cc @DR-Univer

Reply:
Doc also has two services, but docDrawingService will be deprecated in the future.